src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -383,18 +383,35 @@ export default class AircraftController {
     }
 
     /**
+     * Finds an aircraft by its callsign
+     *
      * @method findAircraftByCallsign
      * @param  {string} [callsign='']
      * @return {AircraftModel|null}
      */
     findAircraftByCallsign(callsign = '') {
-        if (callsign === '') {
+        if (!callsign) {
             return null;
         }
 
         return _find(this.aircraft.list, (aircraft) => aircraft.callsign.toLowerCase() === callsign.toLowerCase());
```

Comment:
could you add a `TODO` here to normalize `callsign` casing?  in my opinion, the only thing we should need to lowerCase here would be the passed value

Reply:
addressed below